go.mod

```diff
@@ -53,5 +53,3 @@ require (
 	gopkg.in/ini.v1 v1.61.0
 	gopkg.in/yaml.v2 v2.3.0
 )
-
-replace golang.org/x/sys => golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
```

Comment:
@ceperapl sorry, just saw this. This is required for the Windows build ... at least it used to be.

I don't think that this can be removed ...

Reply:
setting on hold for now that it is not merged accidentially

Reply:
I've returned the go modules in their initial state.

Reply:
Thanks!